warden-roboto/src/main/kotlin/AndroidAttestationChecker.kt

```diff
@@ -149,7 +171,7 @@ abstract class AndroidAttestationChecker(
 
                 root.publicKey.encoded
                 throw CertificateInvalidException(
-                    "No matching root certificate$additionalInfo",
+                    "No matching root certificate$additionalInfo\n  Full chain:\n  ${joinToString("\n    ") { it.encoded.encodeBase64() }}",
```

Comment:
I would not do newlines in exception messages, this messes up the log files ... maybe we can add the certificates in the exception object as members?